_test/common_functions/data_plot_interface_tester.m

```diff
@@ -0,0 +1,27 @@
+classdef data_plot_interface_tester < data_plot_interface
+    %UNTITLED5 Summary of this class goes here
+    %   Detailed explanation goes here
```

Comment:
No explnatory notes.
In particular the tester method setting ndim to 2 when you provide functions for 1-3 dimensions needs a note to understand what this is for